dbsync.sh

```diff
@@ -1,14 +1,14 @@
 #!/bin/bash -e
 
 # Dumps a PostgresSQL over the network, from production to staging
-# Check the internal wiki (http://wiki.scilifelab.se/bioinfo/node/3752) for 
+# Check the internal wiki (http://wiki.scilifelab.se/bioinfo/node/3752) for
 # credentials' information
 
 DB="<insert here the DB name>"
 DB_USER="<insert here the DB user to make the dumping>"
 
-PROD="genologics.scilifelab.se"
-STAGING="genologics-stage.scilifelab.se"
+PROD="ngi-lims-prod.scilifelab.se"
```

Comment:
Does this change apply with immediate effect?

Reply:
No, this scrip is only for documentation if I understand correctly.